MapboxDirections/MBDirections.swift

```diff
@@ -141,10 +142,16 @@ open class Directions: NSObject {
         
         self.accessToken = accessToken!
         
+        if(host?.isEmpty ?? true){
+            self.apiEndpoint = defaultApiEndPointURLString != nil ? URL(string: defaultApiEndPointURLString!)!: URL(string: "https://api.mapbox.com")!
```

Comment:
For clarity, let’s unpack this line into multiple statements that use optional chaining instead of using ternary operators and force unwrapping:

```swift
if let host = host, !host.isEmpty {
    var baseURLComponents = URLComponents()
    // …
    apiEndpoint = baseURLComponents.url!
} else {
    apiEndpoint = URL(string: defaultAPIEndpointString ?? "https://api.mapbox.com")!
}
```